cmd/restic/cmd_check.go

```diff
@@ -346,22 +368,25 @@ func runCheck(ctx context.Context, opts CheckOptions, gopts GlobalOptions, args
 		if err != nil {
 			return err
 		}
+		result.NumUnusedBlobs = len(unused)
```

Comment:
Checking for unused blobs can only be used by tests. So I don't think we should expose that number via the JSON output.

Reply:
Removed this field from the JSON output.